src/include/txn_inline.h

```diff
@@ -312,6 +312,33 @@ __txn_next_op(WT_SESSION_IMPL *session, WT_TXN_OP **opp)
     return (0);
 }
 
+/*
+ * __wt_pending_prepared_next_op --
+ *     Get the next transaction operation slot for a pending prepared transaction.
+ */
+static WT_INLINE int
+__wt_pending_prepared_next_op(
+  WT_SESSION_IMPL *session, WT_TXN_OP **opp, WT_PENDING_PREPARED_ITEM *prepared_item)
+{
+    WT_BTREE *btree;
+    WT_TXN_OP *op;
+
+    *opp = NULL;
+
+    WT_RET(__wt_realloc_def(
+      session, &prepared_item->mod_alloc, prepared_item->mod_count + 1, &prepared_item->mod));
+
+    op = &prepared_item->mod[prepared_item->mod_count++];
+    WT_CLEAR(*op);
+    btree = S2BT(session);
+    op->btree = btree;
+
+    /* Increment the session use count for the data handle. */
+    (void)__wt_atomic_addi32(&session->dhandle->session_inuse, 1);
```

Comment:
Why we need this?

Reply:
it's to increase the reference count of dhandle, maybe we can increment it when we claim prepare?

Reply:
Looks like it's always increase in `__txn_next_op`, and automatically decrease in `__wt_txn_op_free`. If we don't increase here there will be mismatch in decreasing when freeing op.

Reply:
ok, can you add a comment for this?

Reply:
done